api/validators/AdminControllerRequests.ts

```diff
@@ -58,3 +60,19 @@ export class SubmitAttendanceForUsersRequest implements ISubmitAttendanceForUser
   @Allow()
   asStaff?: boolean;
 }
+
+export class UserAccessUpdates implements IUserAccessUpdates {
+  @IsDefined()
+  user: string;
+
+  @IsDefined()
+  @IsIn(['RESTRICTED', 'STANDARD', 'STAFF', 'ADMIN', 'MARKETING', 'MERCH_STORE_MANAGER',
+    'MERCH_STORE_DISTRIBUTOR'])
+  newAccess: string;
```

Comment:
I think this type should be the `UserAccessType` enum itself, that way we don't need to hardcode which exact enum values the string should have (which is susceptible to bugs if we extend the `UserAccessType` enum to hold more values, but don't add that value to this list)

Reply:
also nit: `newAccess` can simply be called `accessType` since it's implied that it is a new access type because the request is for an update